setup.py

```diff
@@ -0,0 +1,18 @@
+# coding: utf-8
+
+from setuptools import setup, find_packages
+
+setup(
+    name='tc_librato',
+    version="0.0.1",
+    description='Thumbor Librato extensions',
+    author='Peter Schröder, Sebastian Eichner',
+    author_email='peter.schroeder@jimdo.com, sebastian.eichner@jimdo.com',
+    zip_safe=False,
+    include_package_data=True,
+    packages=find_packages(),
+    install_requires=[
+        'jimdo-thumbor',
```

Comment:
@heynemann i was trying to reference 23a400b903da78aa9b65f91a1e6231fb294e9a3d from `Jimdo/thumbor` with something like 

```python
    dependency_links=[
        "git://github.com/Jimdo/thumbor.git@23a400b903da78aa9b65f91a1e6231fb294e9a3d#egg=thumbor-jimdo"
    ],
```

but i could not get it working. any ideas?